catweasl.c

```diff
@@ -326,9 +326,9 @@ static void
 CWTriggerStep(catweasel_contr *c)
```

Comment:
This function confuses me on a few points.  I'm not sure if it's working around limitations in the CW hardware and/or is taking some coding shortcuts.

My first point is why it is enforcing a time delay between turning on then off the step bit in the controller.  Is this a bug with the controller where if turning the bit off too quickly it'll fail?

My other is that the fixed, enforced delay in the function seems kludgy.  I would have instead have a "time of last step timestamp" member in the `catweasel_contr` structure.  On entry to this function, read the struct's timestamp and take the difference between the timestamp and "now".  If the difference is less than the step delay, delay for the remaining difference, otherwise, don't wait at all.  After the possible delay check, update the current timestamp then do the step operation.  But this approach also depends on what's up with the reason for the delay between enabling and disabling the controller's step bit (and why it's split evenly).

Changing to use a timestamp isn't necessary for this commit, but it would be nice to be a future enhancement and have the reason stated behind the split delay operation.